src/main/java/com/epam/lowcost/util/Constants.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.lowcost.util;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class Constants {
+    public static int DEFAULT_NUMBER_OF_USERS_ON_PAGE = 5;
```

Comment:
why? you can use only one constant for all entities, they all equal 5...

Reply:
I think its more convenient and readable, as we might want to change default values separately.